internal/api/types.go

```diff
@@ -80,3 +80,41 @@ type listRolesV2Role struct {
 	ID   gidx.PrefixedID `json:"id"`
 	Name string          `json:"name"`
 }
+
+// RoleBindings
+
+type roleBindingResponseRole struct {
+	ID   gidx.PrefixedID `json:"id"`
+	Name string          `json:"name"`
+}
+
+type roleBindingSubjectCondition struct{}
+
+type roleBindingSubject struct {
+	ID        gidx.PrefixedID              `json:"id" binding:"required"`
+	Type      string                       `json:"type,omitempty"`
+	Condition *roleBindingSubjectCondition `json:"condition,omitempty"`
+}
```

Comment:
Let's add conditions once we know what they'll look like. I actually suspect we'll want conditions to be on the binding and not the subject, but we should figure that out.